docs/Makefile

```diff
@@ -77,17 +77,17 @@ qthelp:
 	@echo
 	@echo "Build finished; now you can run "qcollectiongenerator" with the" \
 	      ".qhcp project file in $(BUILDDIR)/qthelp, like this:"
-	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/raven-js.qhcp"
+	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/sentry-js.qhcp"
```

Comment:
Hm, I saw this one but it was copy-pasta'd to this repo and it threw me off. I'll fix it, but I don't think we use that step at all.